FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml

```diff
@@ -264,6 +264,19 @@
                                     <HyperlinkButton Command="{x:Bind AddKeyboardShortcutCommand}" Content="Add" />
                                 </StackPanel>
                                 <views:KeyBindingsView ShowCommandName="False" ViewModel="{x:Bind KeyBindings}" />
+                                <TextBlock Margin="0,0,0,8" Text="Line ending translation on paste" />
```

Comment:
In spirit of the other option titles in the shell profile configuration I would rephrase the text as ⬇️

If you update this, don't forget to update it [here](https://github.com/felixse/FluentTerminal/blob/4349d3ad44169f4de38365a3fc00568f47edbeaa/FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml#L324) as well.

```suggestion
                                <TextBlock Margin="0,0,0,8" Text="Line ending translation" />
```

Reply:
Changed to `Line ending translation mode`